core/src/main/java/org/radargun/sysmonitor/RssMonitor.java

```diff
@@ -0,0 +1,69 @@
+package org.radargun.sysmonitor;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.lang.management.ManagementFactory;
+
+import org.radargun.reporting.Timeline;
+
+public class RssMonitor implements Monitor {
+
+   private static final String RSS_MEMORY_USAGE = "RSS Memory usage";
+   private final Timeline timeline;
+
+   public RssMonitor(Timeline timeline) {
+      this.timeline = timeline;
+   }
+
+   @Override
+   public void start() {
+   }
+
+   @Override
+   public void stop() {
+   }
+
+   @Override
+   public void run() {
+      Process process = null;
+      try {
+         process = createProcess();
+         Long rssUsage = getRssUsageFrom(process);
+         timeline.addValue(Timeline.Category.sysCategory(RSS_MEMORY_USAGE), new Timeline.Value(rssUsage));
+      } finally {
+         if (process != null) process.destroy();
+      }
+   }
+
+   private Process createProcess() {
+      Process process;
+      try {
+         String command = "ps --no-headers -o rss " + getCurrentPid();
+         ProcessBuilder builder = new ProcessBuilder().command("bash", "-c", command);
```

Comment:
Just wondering if this works without a full path to `bash`? Adding the path will make it even less portable though!